src/screens/benefit/Details.tsx

```
@@ -114,23 +112,67 @@ const BenefitsDetails: React.FC = () => {
+		} catch (err) {
+			console.error('Error in checking eligibility', err);
+			setError('Failed to check eligibility. Please try again later.');
+			setLoading(false);
+			return;
+		}
+
+		// Step 2: Process eligibility reasons
+		const reasons =
+			eligibilityResponse?.ineligible?.[0]?.details?.reasons || [];
+
+		const reasonMessages = reasons.map((r: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)